rest/dbHandler.go

```diff
@@ -127,7 +114,26 @@ func (dbHandler DbHandler) deleteTrackPage(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	// Send response
-	w.WriteHeader(c)
+	w = prepareGeneralResponse(w, []byte(trackID), constant.TextPlain, nil, "deleteTrackPage")
+	return
+}
+
+// Prepares a general response to send back to the client, setting various common variables in the ResponseWriter
+func prepareGeneralResponse(unPrepW http.ResponseWriter, rawData []byte, contentType string, jsonError error, callerFunc string) (prepW http.ResponseWriter) {
+	prepW = unPrepW
+	if jsonError != nil {
+		gtbackend.DebugLog(gtbackend.InternalLog{
+			Origin: fileNameDB,
+			Method: callerFunc + "-prepareGeneralResponse",
+			Err:    jsonError,
+		})
+
+		http.Error(prepW, jsonError.Error(), http.StatusBadRequest)
+		return
+	}
+
+	prepW.Header().Set("Content-Type", contentType)
```

Comment:
Just a note, I think we have a const of "Content-Type"

Reply:
Oh yeah we do.